contract/src/main/kotlin/io/provenance/scope/loan/utility/ContractRequirements.kt

```diff
@@ -0,0 +1,86 @@
+package io.provenance.scope.loan.utility
+
```

Comment:
The goal of this file was to provide a super flexible and type-safe implementation of enforcing boolean data requirements however a contributor would like to.

Reply:
:dank: